### PR TITLE
all: use yield from

### PR DESCRIPTION
**Pull request by birkenfeld**

### CODE REVIEW

File: pygments/lexers/scripting.py [hunk 1]
```diff
@@ -1253,13 +1253,13 @@ class MiniScriptLexer(RegexLexer):
...
-            	'sort', 'shuffle', 'push', 'pop', 'pull', 'range',
-            	'print', 'input', 'time', 'wait', 'locals', 'globals', 'outer',
-            	'yield'), suffix=r'\b'),
+                'abs', 'acos', 'asin', 'atan', 'ceil', 'char', 'cos', 'floor',
+                'log', 'round', 'rnd', 'pi', 'sign', 'sin', 'sqrt', 'str', 'tan',
+                'hasIndex', 'indexOf', 'len', 'val', 'code', 'remove', 'lower',
+                'upper', 'replace', 'split', 'indexes', 'values', 'join', 'sum',
+                'sort', 'shuffle', 'push', 'pop', 'pull', 'range',
+                'print', 'input', 'time', 'wait', 'locals', 'globals', 'outer',
+                'yield'), suffix=r'\b'),
```
Comment (Anteru) [inline pygments/lexers/scripting.py:1262]: I assume this is a drive-by shooting that the formatting changed here?

File: pygments/lexers/sql.py [hunk 2]
```diff
@@ -117,9 +117,7 @@ class PostgresBase:
     def get_tokens_unprocessed(self, text, *args):
         # Have a copy of the entire text to be used by `language_callback`.
         self.text = text
-        for x in super(PostgresBase, self).get_tokens_unprocessed(
-                text, *args):
-            yield x
+        yield from super(PostgresBase, self).get_tokens_unprocessed(text, *args)
```
Comment (Anteru) [inline pygments/lexers/sql.py:120]: `yield from super().`?

File: pygments/lexers/scripting.py (hunk 1 above)
Comment (birkenfeld) [reply pygments/lexers/scripting.py:1262]: yes

File: pygments/lexers/sql.py (hunk 2 above)
Comment (birkenfeld) [reply pygments/lexers/sql.py:120]: next PR :)

File: pygments/lexers/zig.py [hunk 3]
```diff
@@ -122,8 +122,3 @@ class ZigLexer(RegexLexer):
             (r'"', String, '#pop')
         ]
     }
-
-    def get_tokens_unprocessed(self, text):
-        for index, token, value in \
-                RegexLexer.get_tokens_unprocessed(self, text):
-            yield index, token, value
```
Comment (Anteru) [inline pygments/lexers/zig.py:129]: Why was this removed wholesale? I presume because that's equivalent to the parent class implementation?

Comment (birkenfeld) [reply pygments/lexers/zig.py:129]: Since the parent class is RegexLexer, AFACIS it just delegates everything to it.

**Review by Anteru — APPROVED** [review]
Looks good to me, but I do have questions.
